src/components/FAQ.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+
+interface FAQProps {
+  question: string;
+  answer: string;
+}
+
+export default function FAQ(props: FAQProps) {
+  const [show, setShow] = useState(false);
+
+  return (
+    <div className='c-faq'>
+      <div onClick={() => setShow(!show)} className='c-faq__question'>
+        <h4>{props.question}</h4>
+        {show ? <UpArrow /> : <DownArrow />}
+      </div>
+      {show ? (
+        <div className='c-faq__answer'>
+          <h4>{props.answer}</h4>
+        </div>
+      ) : (
+        ""
+      )}
+    </div>
+  );
+}
+
+const UpArrow = () => {
```

Comment:
You can create an svg file and import is as a normal react component
See how I used it here
<img width="653" alt="Screenshot 2023-08-23 at 22 13 27" src="https://github.com/geektutor/devfestlagos2023-web/assets/36075575/cd5aa103-a27a-47d4-9e53-c65013b6ce7a">

<img width="665" alt="Screenshot 2023-08-23 at 22 13 35" src="https://github.com/geektutor/devfestlagos2023-web/assets/36075575/7dc7ec13-3360-4563-84cd-babb72a83a4e">